### ISSUE TITLE
Hardcoded grey / gray color of description preview text doesn't play well with selected rows or themes

### ISSUE THREAD

**Issue by DonQuixoteI** [getting-things-gnome/gtg#576]
If the task is selected in the task browser, the grey color of description preview is difficult to read on top of blue background in the default white mode.

It will be easier to read if the description preview and the task counters in the sidebar (same permanent grey color) will have the same colors as tasks, tags and dates: black if not selected and white if selected.

The readability of description (for selected task) is a bit better in the dark mode, but the black icons are very difficult to see in case of dark grey background of dark mode.

**Comment by nekohayo** [comment]
Is it hardcoded as grey in the code, or is it using transparency? (if possible, I would think transparency would work in all cases?)

I'd prefer automatic solutions (not options), if transparency can be used when not selected and full opacity when the row is selected... but then @diegogangl wanted to redo the whole widgetry so unless there's a trivial fix it would probably be moot to address that before then...
